src/lib/services/winners-service.ts

```diff
@@ -3,23 +3,74 @@ import { PockyDB } from '../database/db-interfaces';
 import TableHelper from '../parsers/tableHelper';
 import __logger from '../logger';
 import { Receiver } from '../../models/receiver';
+import Config from '../config';
 
 export interface WinnersService {
+	getWinners(results: ResultRow[]): ResultRow[]
 	returnWinnersResponse(): Promise<string>
 }
 
 export class DefaultWinnersService implements WinnersService {
 	readonly cannotDisplayResults: string = 'Error encountered; cannot display winners.';
 	database : PockyDB;
+	config : Config;
 
-	constructor(database : PockyDB) {
+	constructor(database : PockyDB, config: Config) {
 		this.database = database;
+		this.config = config;
+	}
+
+	pegValid(comment: string, requireKeywords: number, keywords: string[], penaltyKeywords: string[]): boolean {
```

Comment:
This is essentially a duplicate of Utilities.commentIsPenalty.

Reply:
Almost. This method does take into account whether keywords are required. That said, I have moved this method into utilities

Reply:
Yeah to be honest the method that was already there probably should have taken that into account as well.